src/tests/test_config.py

```diff
@@ -130,11 +167,11 @@ def test_magic_initialization_when_default_connection_fails(
     assert "Cannot start default connection" in captured.out
 
 
-def test_magic_initialization_with_no_pyproject(tmp_empty, ip_no_magics):
+def test_magic_initialization_with_no_toml(tmp_empty, ip_no_magics):
     load_ipython_extension(ip_no_magics)
 
 
-def test_magic_initialization_with_corrupted_pyproject(tmp_empty, ip_no_magics, capsys):
+def test_magic_initialization_with_corrupted_toml(tmp_empty, ip_no_magics, capsys):
```

Comment:
we can add a similar test for the alternate config also

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/899/files#diff-c7a5329ac9b1ee1d1f5f3018031825611df32db9214cb42981f95243bdbd400bR188-R206)